plugins/services/src/js/components/modals/NewCreateServiceModalForm.js

```diff
@@ -197,9 +179,12 @@ class NewCreateServiceModalForm extends Component {
   }
 
   getAppConfig(currentState = this.state) {
-    let {baseConfig, batch} = currentState;
-    let patch = batch.reduce(jsonConfigReducers, {});
-    return CreateServiceModalFormUtil.applyPatch(baseConfig, patch);
+    let {batch} = currentState;
+    return batch.reduce(this.props.jsonConfigReducers, {});
+    // aplying patch causes duplication of key-value structures like Lables, ENV and secrets
+    // let {baseConfig, batch} = currentState;
```

Comment:
Let's please address this issue instead of simply removing the feature - @wavesoft might has an idea how to fix it...